lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,27 @@
+import { s3PutObject } from '@cumulus/aws-client/S3';
+
+/**
+ * Store migration errors JSON file on S3.
+ *
+ * @param {string} bucket
+ * @param {string[]} message
+ * @param {string} recordClassification
+ * @param {string | undefined} stackName
+ * @returns {void}
+ */
+export const storeErrors = async (
+  bucket: string,
+  message: string[],
+  recordClassification: string,
+  stackName?: string) => {
+  const file = `{
+    "errors": ${JSON.stringify(message)}
+  }`;
+  const filename = `data-migration2-${recordClassification}-errors.json`;
```

Comment:
I think that the filename should be timestamped, so that each run of the migration has a separate error log that you can refer to.